src/pages/HealthContent.tsx

```diff
@@ -1,110 +1,50 @@
-import React from "react";
-// @ts-ignore
-import workoutImg from "../assests/images/workout.jpg";
+import React, { useState, useEffect } from "react";
 
 import "react-multi-carousel/lib/styles.css";
 import SingleCarousel from "../components/SingleCarousel";
+import { CardHomeInterface as CardInterface } from "../interfaces/CardHomeInterface";
 import styles from "./HealthContent.module.css";
 
 const HealthContent = () => {
-  const cards = [
-    {
-      id: 1,
-      title: "1",
-      description:
-        "Managing and creating weekly workout plans for your convenience",
-      images: workoutImg,
-      link: "workouts",
-    },
+  const [pageCards, setPageCards] = useState<CardInterface[]>();
 
-    {
-      id: 2,
-      title: "2",
-      description:
-        "Managing and creating weekly workout plans for your convenience",
-      images: workoutImg,
-      link: "workouts",
-    },
+  useEffect(() => {
+    fetchData();
+  }, []);
 
-    {
-      id: 3,
-      title: "3",
-      description:
-        "Managing and creating weekly workout plans for your convenience",
-      images: workoutImg,
-      link: "workouts",
-    },
+  const fetchData = async function () {
+    try {
+      const res = await fetch("http://localhost:3000/health-content");
```

Comment:
you could probably create a constandts file where you put configuration data strings and such like this url